.github/workflows/cla_checker.yaml

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I would rather this to be added to the DPW instead

Reply:
I am discussing that with @carlcsaposs-canonical

Reply:
I would argue - that for all DP repos to have the same set of checks that this should be in DPW regardless. Similar to the `lint` step